### DISCUSSION TITLE
Are there any projects developed based on Lexical?

### DISCUSSION THREAD

**Discussion by Darcrandex** [facebook/lexical#5101] [Q&A]
Learning from each other

**Comment by anteloio** [comment]
The only one I've seen so far is [Verbum](https://github.com/ozanyurtsever/verbum), but I'd also be interest to see other examples.

**Comment by moy2010** [comment]
On the discord channel there are several examples: https://discord.com/channels/953974421008293909/964928369793843230

**Reply by anteloio** [reply]
Weird do we need any specific role to access this channel?

<img width="707" alt="image" src="https://github.com/facebook/lexical/assets/1740848/80884e70-3c09-42a6-8875-b50c14189aa1">

**Reply by moy2010** [reply]
Not at all. Try with this invitation link: https://discord.gg/wRra7D45sm

**Reply by anteloio** [reply]
It works! Thanks 🙏

**Comment by fdemian** [comment]
I have an [editor](https://github.com/fdemian/kalliope) that's based on lexical, which I'm using for other projects (which are private for now, although one of them is about to become public).
